### ISSUE TITLE
Precious? General? Who knows. M+ channels are TOO spammy.

### ISSUE THREAD

**Issue by Wanhedaa** [Icecrown-Community/Icecrown-Precious-Bot-Public#115]
The whole channel visiblity was because the new boosters want to see, if there is any life in this community. Don't we have an "archived m+ runs" channel, where the already DONE and submited runs go? Or can't we have a channel where precious can spamm the done runs? 

ATM as an advertiser you can see this 
![image](https://user-images.githubusercontent.com/77633500/136964905-ee45f9b3-865f-4d01-ba8c-bf200fa74770.png)

Which is contains everyone's run. So if you have multiple runs as an advertiser, you have to click 1 by 1 each completed run to check if its yours or not (ofc you could do it via Precious, but then again, legendary tickets spamm us so we might miss some runs).

As a booster and advertiser you get spamm pinged even if its not relevant anymore. Before if the run is gone to running phase, the ping dissapeard so it didn't disturb anyone. 

TLDR: Get a "keys submited" channel where every run goes after completed phase, and make it visible to everyone (without pinging any role) and make the running / completed categorys invisible

**Comment by Bgabor997** [comment]
implemented and deployed 2021.10.14
